assets/js/src/core/modules/asset/editor/types/folder/tab-manager/tabs/list/sidebar/filters/filter-container-inner.tsx

```diff
@@ -72,39 +106,101 @@ export const FilterContainerInner = (): React.JSX.Element => {
        }
     >
       <Content padded>
-        <Title>Search & Filter</Title>
-
-        <Form>
-          <Space
-            direction='vertical'
-            style={ { width: '100%' } }
-          >
-            <Search
-              placeholder='Search'
-              value={ '' }
+        <Flex
+          align='center'
+          justify='space-between'
+        >
+          <Title>Search & Filter</Title>
+          <Flex gap='extra-small'>
+            <Text>Advanced Mode</Text>
+            <Switch
+              checked={ isAdvancedMode }
+              onChange={ () => { setIsAdvancedMode(!isAdvancedMode) } }
             />
+          </Flex>
+        </Flex>
 
-            <Checkbox
-              checked={ isIncludeDescendants }
-              onChange={ handleChangeIsIncludeDescendants }
+        {isAdvancedMode
+          ? (
+            <Flex
+              gap='extra-small'
+              vertical
             >
-              only direct children
-            </Checkbox>
-
-            {/* <Checkbox */}
-            {/*  checked={ false } */}
-            {/*  value={ 'referenced' } */}
-            {/* > */}
-            {/*  only unreferenced */}
-            {/* </Checkbox> */}
-          </Space>
-        </Form>
-
-        <Title>
-          Field filters
-        </Title>
-
-        <FieldFiltersContainer />
+              <Flex gap='mini'>
```

Comment:
Can we create a stand alone component for this PQL Filter/Input? 
At a later state of the project, we will use this component in several places and bundles. 

And also the idea is to integrate some syntax highlighting in the future. 
What would also imply to move this to a separate component. :)